docs/usage.rst

```diff
@@ -142,6 +142,38 @@ separate functions based on where the model is being run.
 .. autofunction:: fv3config.run_native
 .. autofunction:: fv3config.run_docker
 
+Submitting a Kubernetes job
+---------------------------
+
+A python interface is provided for submitting `fv3run` jobs to Kubernetes. Here's
+an example for submitting a job based on the default configuration dictionary::
+
+    import yaml
+    import gcsfs
+    import fv3config
+
+    config_location = 'gs://my_bucket/fv3config.yml'
+    outdir = 'gs://my_bucket/rundir'
+    docker_image = 'us.gcr.io/vcm-ml/fv3config-python'
```

Comment:
This image has not been pushed to GCS, so I couldn't test whether this worked or not.

Reply:
Typo has been fixed, it's `fv3gfs-python`.